contract-tests/v3_api/conftest.py

```diff
@@ -1,4 +1,26 @@
-import urllib3
-
-# Disable any warnings about SSL connections
-urllib3.disable_warnings()
+import pytest
+import subprocess
+
+from faker import Faker
+
+
+@pytest.fixture(scope="session")
+def headers():
+    # Create a test user
+    fake = Faker()
+    email = fake.company_email()
+    user = fake.user_name()
+
+    # Add them as a superuser to the system running in Docker
+    docker_compose = (
+        subprocess.check_output("which docker-compose", shell=True).decode("ascii").strip("\n")
+    )
+    subprocess.call(
+        "{} run app python manage.py createsuperuser --noinput --email={} --user={}".format(
+            docker_compose, email, user
+        ),
+        shell=True,
+    )
```

Comment:
I think we can probably do this inside the circleci config rather than here?

Reply:
Yeah, it is just a command to create the super-user the test needs. Unless you want to create a specific superuser just for CI and have it always available?

Reply:
Like can't you just move these steps here:

https://github.com/mozilla/normandy/pull/2132/files#diff-1d37e48f9ceff6d8030570cd36286a61R257-R261